### DISCUSSION TITLE
installed gmxmmpbsa

### DISCUSSION THREAD

**Discussion by AbdelmoujoudFaris** [Valdes-Tresanco-MS/gmx_MMPBSA#341] [General]
Dear, I hope this message finds you well. I am writing to request your assistance with installing GMXMMPBSA on an HPC. I have attempted to install the software myself, but have encountered some issues that I have been unable to resolve.

If possible, I would greatly appreciate your guidance on the installation process and any troubleshooting tips you may have. I am also happy to provide any additional information or log files that may be helpful in diagnosing the problem.

Thank you for considering my request. I look forward to hearing from you soon.

Best regards,

**Comment by marioernestovaldes** [comment]
I have opened a new issue with your inquiry, so it's easier for us to keep track of it... please, let's continue at https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/issues/342

Please submit any log file you have from the installation process to see what's the error. @Valdes-Tresanco-MS will guide you throughout the process. We have had gmx_MMPBSA successfully installed in HPC systems before.
